core/src/com/group66/game/cannon/Cannon.java

```diff
@@ -137,14 +137,8 @@ public void update(float delta) {
      * @param batch the batch used to draw with
      */
     public void draw(SpriteBatch batch) {
-        //batch.draw(cannon_texture, hitbox.x, hitbox.y, BALL_RAD, BALL_RAD);
-
-        batch.draw(cannonTextureRegion, xpos - width / 2f - 3, ypos - height / 2f + 23, 
-                width / 2f, height / 2f, width, height, 5 / 2, 1, angle, true);
-
-        //batch.draw(cannon_texture_region, x - width / 2f, y - height / 2f,    
-        //width / 2f, height / 2f, width, height, 1, 1, angle, true);
-
+        batch.draw(cannonTextureRegion, xpos - width / 2f + 11, ypos - height / 2f + 7, 
```

Comment:
I guess the best option then is to change the texture sprite so the rotational center is in the center, this makes it so we can just use another cannon sprite without changing the code. (We could just add transparent pixels at the bottom or top wherever needed in the image to make it centered)